discovery.md

```diff
@@ -110,6 +117,76 @@ be considered an event subscriber and a target user of this API.
 This API is specified as a REST API with well defined entities and relationships
 between those entities.
 
+To help explain the data model, the following non-normative pseudo yaml shows
+its basic conceptual structure:
+
+( `*` means zero or more, `+` means one or more, `?` means optional)
+
+```
+- services: *
+  - service: [unique identifier for producer of the specified event types]
+    description: [human string] ?
+    uri: [URI reference for human documentation] ?
+    specversion: [ce-speciversion value] ? (required?)
+    subscriptionuri: [URI to which the Subscribe request will be sent]
+    subscriptionconfig: *
+    authscope: [string] ?  (explain)
```

Comment:
I would like to see something like `sourcepattern` in here. @mikehelmick had this in an earlier proposal.

Reply:
I assume if we added that we'd then need to find a way to define all of the various substitution values, right? 
Do you have a link to his old proposal?

Reply:
What I have in mind is similar to what AsyncAPI offers for channels:
[https://www.asyncapi.com/docs/specifications/2.0.0/#a-name-parametersobject-a-parameters-object](https://www.asyncapi.com/docs/specifications/2.0.0/#a-name-parametersobject-a-parameters-object
)

In the original CloudSubscriptions Google doc, you can find:
> “sourceStructure”
Documentation on the format of the source, for example
“github.com/[organization]/[repository]/pull/[id]

Reply:
@deissnerk want to open an issue so we can track your idea?